src/components/CommentList/CommentList.tsx

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import { Comment } from '../../react-app-env';
 import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comment: Comment | undefined;
+}
+
+export const CommentList: React.FC<Props> = ({ comment }) => (
+  <CommentInfo comment={comment} />
```

Comment:
here you should map through comments array and create a CommentInfo component for each one